lib/entities/organization-invitation.ts

```diff
@@ -0,0 +1,30 @@
+import cloneDeep from 'lodash/cloneDeep'
+import { freezeSys, toPlainObject } from 'contentful-sdk-core'
+import { AxiosInstance } from 'axios'
+import { MetaLinkProps, DefaultElements } from '../common-types'
+
+export type OrganizationInvitationProps = {
+  firstName: string
```

Comment:
Is there a `sys`?

Reply:
I am not really sure how to handle this one...
<img width="761" alt="Screenshot 2020-06-08 at 11 50 57" src="https://user-images.githubusercontent.com/4272331/84021006-1ddcac80-a984-11ea-9091-a4f18dc234c7.png">

Reply:
```
sys: MetaSysProps & { user: User | null, invitationUrl: string  }
```

Reply:
@suevalov updated - one thing that we talked about is user type - we don't have any `User` in common types, maybe I would create one? I am not really sure if we gonna need it in many places though..

Reply:
if the shape is consistent for all usages then it probably makes sense to create, otherwise, let's leave it here